core/arch/arm/plat-imx/main.c

```diff
@@ -189,6 +189,8 @@ void plat_cpu_reset_late(void)
 #if defined(CFG_BOOT_SYNC_CPU)
 		psci_boot_allcpus()
 #endif
-		imx_configure_tzasc();
+#if defined(CFG_TZC380)
+	imx_configure_tzasc();
```

Comment:
minor: indentation.

Reply:
Addressed in a6bca6b924766606e377fc187444f92ce92122dc